extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
```

Comment:
Why do you need an own bundle for the ClassicUI Icon set? Can´t it be put into the ClassicUI bundle? And does it really contain 4 Icons only?